query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/ExecutionContextTest.java

```diff
@@ -159,13 +159,11 @@ public void testReferencedColumns() {
             .setRhs(expression)
             .setOperator(Operator.EQ);
     Filter startTimeFilter =
-        QueryRequestBuilderUtils.createTimeFilter(
+        createTimeFilter(
             "Trace.start_time_millis",
             Operator.GT,
-            startTimeInMillis - Duration.ofHours(24).toMillis());
-    Filter endTimeFilter =
-        QueryRequestBuilderUtils.createTimeFilter(
-            "Trace.end_time_millis", Operator.LT, startTimeInMillis);
+            endTimeInMillis - Duration.ofHours(24).toMillis());
```

Comment:
nit: calculating end time from start time is safer than calculating start time from end time, since there's less potential for ending up with a negative time.